requirements.txt

```diff
@@ -6,6 +6,12 @@ jsonschema==3.2.0
 lz4==3.0.2
 requests==2.23.0
 networkx==2.5
+six~=1.15.0
```

Comment:
Is this required by some of the dependencies? If not it should be removed

Reply:
i got protobuf_to_dict in some project written in python2.7 and it require six

Reply:
i will try get rid of it

Reply:
done